packages/eslint-config-react-app/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
We'll also want a readme here, explaining in a few words how to use this package as standalone.